packages/link-modules/src/services/link-module-service.ts

```diff
@@ -256,14 +303,29 @@ export default class LinkModuleService<TLink> implements ILinkModule {
       )
     }
 
+    await this.eventBusModuleService_?.emit<Record<string, unknown>>(
+      (deletedEntities as { id: string }[]).map(({ id }) => ({
+        eventName: this.entityName_ + ".dettached",
+        body: {
+          metadata: {
+            service: this.serviceName_,
+            action: "dettached",
```

Comment:
todo: Typo, perhaps these actions should be "constified"
```suggestion
        eventName: this.entityName_ + ".detached",
        body: {
          metadata: {
            service: this.serviceName_,
            action: "detached",
```